vnext/ReactUWP/Views/SliderViewManager.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include <Views/ShadowNodeBase.h>
+#include "SliderViewManager.h"
+
+#include <Utils/ValueUtils.h>
+
+#include <IReactInstance.h>
+
+#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
+
+namespace winrt {
+using ToggleButton = Windows::UI::Xaml::Controls::Primitives::ToggleButton;
+}
+
+namespace react {
+namespace uwp {
+
+class SliderShadowNode : public ShadowNodeBase {
+  using Super = ShadowNodeBase;
+
+ public:
+  SliderShadowNode() = default;
+  void createView() override;
+  void updateProperties(const folly::dynamic &&props) override;
+};
+
+void SliderShadowNode::createView() {
+  Super::createView();
+
+  auto slider = GetView().as<winrt::Slider>();
+  auto wkinstance = GetViewManager()->GetReactInstance();
+}
+
+void SliderShadowNode::updateProperties(const folly::dynamic &&props) {
+  m_updating = true;
```

Comment:
>m_updating [](start = 2, length = 10)

why do you need m_updating flag?

Reply:
All the shadow nodes do this, i couldn't tell you why

---
In reply to: [335127109](https://github.com/microsoft/react-native-windows/pull/3400#discussion_r335127109) [](ancestors = 335127109)